plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractIcebergTableOperations.java

```diff
@@ -257,4 +257,26 @@ protected static List<Column> toHiveColumns(List<NestedField> columns)
                         Optional.empty()))
                 .collect(toImmutableList());
     }
+
+    protected boolean isCommitSuccess(String metadataLocation)
+    {
+        AtomicReference<Boolean> isSuccess = new AtomicReference<>(false);
```

Comment:
Why atomic when it runs in one thread?

Reply:
I took the reference from https://github.com/apache/iceberg/blob/3cddc9f28c93b9231060ecb6b90e2d524bd5d160/core/src/main/java/org/apache/iceberg/BaseMetastoreTableOperations.java#L304